### PR TITLE
Add iCat Game

### PR DESCRIPTION
**Pull request by kongtaoxing**

### CODE REVIEW

**Comment by eherrerosj** [comment]
hey @kongtaoxing, could you also add "iCat Game" to the list of sub_ecosystems under the Arbitrum eco file (arbitrum.toml)?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a comment

File: data/ecosystems/a/arbitrum.toml [hunk 1]
```diff
@@ -22,6 +22,7 @@ sub_ecosystems = [
   "GMX",
   "Gravita Protocol",
   "Highstreet Market",
+  "iCat Game",
```
Comment (kongtaoxing) [inline data/ecosystems/a/arbitrum.toml:25]: Do you mean modify the file like this, sir?

Comment (eherrerosj) [reply data/ecosystems/a/arbitrum.toml:25]: yep, thx

**Review by eherrerosj — APPROVED** [review]

**Pull request merged**